README.md

```diff
@@ -1,62 +1,82 @@
-
-There are instructions for setup on various platforms, with particular support for Mac and Ubuntu, which can be found here:
-- [Mac Install Walkthrough][mac-installation]
-- [Ubuntu Install Walkthrough][ubuntu-installation]
-
-If you prefer to isolate your install in a virtual machine, you may find it simpler to setup using Vagrant:
-- [Vagrant installation][vagrant-installation]
-
-We don't promise support for Windows, but at one point we had it running and we've kept those docs available for reference
-- [Outdated Windows Walkthrough][windows-installation]
-
-## Licensing information
-
-This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or(at your option) any later version.
-
-This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for more details.
-
-The license can be read [here][license].
-
-Copyright (c) 2017 Connor Turland
-
-[site-beta]: http://metamaps.cc
-[license]: https://github.com/metamaps/metamaps/blob/develop/LICENSE
-[contributing]: https://github.com/metamaps/metamaps/blob/develop/doc/CONTRIBUTING.md
-[contributing-issues]: https://github.com/metamaps/metamaps/blob/develop/doc/CONTRIBUTING.md#reporting-bugs-and-other-issues
-[mac-installation]: https://github.com/metamaps/metamaps/blob/develop/doc/MacInstallation.md
-[ubuntu-installation]: https://github.com/metamaps/metamaps/blob/develop/doc/UbuntuInstallation.md
-[vagrant-installation]: https://github.com/metamaps/metamaps/blob/develop/doc/VagrantInstallation.md
-[windows-installation]: https://github.com/metamaps/metamaps/blob/develop/doc/WindowsInstallation.md
+[![Build Status](https://travis-ci.org/metamaps/metamaps-ui.svg?branch=master)](https://travis-ci.org/metamaps/metamaps-ui)
+
+Make sure you're running a good up to date LTS version of `node`, like 8.9.4
+
+Make sure you have `node-sass` installed
+`$ npm install -g node-sass`
+
+Run the following at the same time, in TWO SEPARATE terminals. We tell the server where the backend process is running with the API environment variable
+JS files, and CSS will rebuild automatically, just refresh the page
+If coding the server itself, you will have to use nodemon, or kill and restart the server process manually
+```
+$ API=http://localhost:3001 node server.js
+$ node-sass -w sass/application.scss public/css/application.css
+```
+
+To make sure the css files get built, use the following in another terminal
+```
+touch sass/application.scss
+```
+
+Run the metamaps api in another terminal using (on port 3001, so the UI can talk to it)
```

Comment:
yeah, I agree with keeping the history, but I think going with two repos is the way, since what we are starting to envision is the possible evolution away from the rails backend